sketch.js

```diff
@@ -1,18 +1,29 @@
 var amplitude, 
     level,
     song, 
-    speed;
+    speed,
+    amplitudeLevel,
+    count = 0;
 
 function preload(){
   song = loadSound('/assets/FortunateSon.mp3');
 }
 
 function setup() {
-  createCanvas(0, 0)
+  createCanvas(0, 0);
   song.play();
-  peaks = song.getPeaks([width]);
   amplitude = new p5.Amplitude();
-  level = amplitude.getLevel();
+
 }
 
-module.exports = level;
\ No newline at end of file
+function draw() {
+  level = amplitude.getLevel();
+  var size = map(level, 0, 1, 1, 30);
+  count++;
+  if (count === 1) {
+    amplitudeLevel = createElement('span', size).addClass('amplitudeLevel');
+  } else {
+    amplitudeLevel.html('');
```

Comment:
If you don't clear the amplitudeLevel from the dom, it keeps creating seemingly infinite elements in the loop with this id. Thereby, I clear it and then reset it. 

Need if statement because you can't clear nothing on the first go or you will get undefined.